tests/jballerina-unit-test/src/test/resources/test-src/annotations/external_dependency_annotation_negative.bal

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+// Types not allowed as sources for ExternalDependency annotation
```

Comment:
Do we need this comment? shall we remove? I did not get the comment as well.

Reply:
Fixed in 6d3dd2d12312dc2613b3dc83b67433e633510f18.